### PR TITLE
ci: standardize testpaths and fail fast

### PR DESCRIPTION
**Pull request by RogerHYang**
add `-x` to fail fast
https://docs.pytest.org/en/8.3.x/reference/reference.html#command-line-flags

### CODE REVIEW

File: pyproject.toml [hunk 1]
```diff
@@ -123,6 +123,8 @@ container = [
   "strawberry-graphql[opentelemetry]==0.236.0",  # need to pin version because we're monkey-patching
   "uvloop; platform_system != 'Windows'",
 ]
+test = [
+]
```
Comment (axiomofjoy) [inline pyproject.toml:127]: Still need this extra?

File: pyproject.toml [hunk 2]
```diff
@@ -249,6 +251,7 @@ addopts = [
   "-rA",
   "--import-mode=importlib",
   "--doctest-modules",
+  "--new-first",
```
Comment (axiomofjoy) [inline pyproject.toml:254]: This run brand-new tests first?

**Review by axiomofjoy — APPROVED** [review]

File: pyproject.toml (hunk 1 above)
Comment (RogerHYang) [reply pyproject.toml:127]: just to suppress the warning if we move to tox

File: pyproject.toml (hunk 2 above)
Comment (RogerHYang) [reply pyproject.toml:254]: yes, this is just for local development, it sorts them by modification date

Comment (RogerHYang) [reply pyproject.toml:254]: on github there's no modification time

**Pull request merged**